sdk/synapse/Azure.Analytics.Synapse.Artifacts/src/Generated/DataFlowRenameDataFlowOperation.cs

```diff
@@ -26,17 +26,20 @@ protected DataFlowRenameDataFlowOperation()
 
         internal DataFlowRenameDataFlowOperation(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, Request request, Response response)
         {
-            _operation = new OperationInternals(clientDiagnostics, pipeline, request, response, OperationFinalStateVia.Location, "DataFlowRenameDataFlowOperation");
+            var nextLinkOperation = NextLinkOperationImplementation.Create(pipeline, request.Method, request.Uri.ToUri(), response, OperationFinalStateVia.Location);
```

Comment:
nit: can we replace `var` with the type here?

Reply:
Fixed in autorest.csharp repo